### ISSUE TITLE
Suggestions : strike, recently used Notebook, icons

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fracte/keepnote#9]
```
One feature that could be useful and probably easy to implement is the
addition  of a "strike" button/function.
Use case : In a "Todo list", it's usefull to remember what's is done rather
than supperess the entry.

A submenu with the last used Notebook is a feature that I guess that most
of the users will be happy with. The possibility to loade automatically the
last used Notebook could be also interesting.

"Bells and whistles" the possibility to associate an icon with a folder.




```

Original issue reported on code.google.com by `Patrick....@gmail.com` on 16 Feb 2009 at 7:56

**Comment by GoogleCodeExporter** [comment]
```
N.B. I have not find the way to classify this to enhancement category :-(
```

Original comment by `Patrick....@gmail.com` on 16 Feb 2009 at 7:58

**Comment by GoogleCodeExporter** [comment]
```
Hi Patrick.  Strikes will be available in the next release.  I also like your
recently open notebooks idea.  That should not be too hard to implement.  
Changing
icons for notes and folders will also most likely be in the next release.  
Thanks for
your feedback.
```

Original comment by `matt.rasmus@gmail.com` on 23 Feb 2009 at 3:35
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `matt.rasmus@gmail.com` on 23 Feb 2009 at 3:36
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Strikes are now available in 0.5.2.  Expect to see recent notebooks in a future 
release.
```

Original comment by `matt.rasmus@gmail.com` on 14 Apr 2009 at 1:30
- Changed state: **Done**
